assets/css/place-row.scss

```diff
@@ -94,8 +94,14 @@
     }
   }
 
-  .place-row__mode-line-icon:not(:last-child) {
-    margin-right: 8px;
+  .place-row__mode-line-icon {
+    &--blue,
+    &--bus {
+      box-shadow: 0px 0px 2px 0px #2e3e4d66;
+    }
```

Comment:
I'm not sure this is doing anything? I tried toggling it and see no changes.

Reply:
The drop shadow was originally added to the SVG itself, but I was seeing some issues with the pills that were not present in pills w/o drop shadow. I tried out the original SVGs with the drop shadow again just in case and they seem good to me. I must have made another change along the way that corrected the original problem. Do you mind pulling it up and checking that BL and Bus pills look good on your machine?

Reply:
👍 all route pills look good